runtime/index.ts

```diff
@@ -1,3 +1,2 @@
 export * from './bootstrapper';
 export * from './config';
-export * from './runtime.module';
```

Comment:
No one should be using the runtime module (we left it in for backwards compatibility with SKY UX 2).